res/values-in/strings.xml

```diff
@@ -53,8 +53,8 @@
     <string name="SERVER">"SERVER"</string>
     <string name="Joining_game_in_">"Bergabung permainan di "</string>
     <string name="Game_Full">"Permainan Penuh"</string>
-    <string name="US_East">"Amerika Tengah"</string>
-    <string name="US_West">"Amerika Utara"</string>
+    <string name="US_East">"Amerika Utara"</string>
+    <string name="US_West">"Amerika Barat"</string>
```

Comment:
Wrong changes.

`US_East`: It should keep as "Amerika Tengah".
`US_West`: It should keep as "Amerika Utara.

You literally inverted and returned the old names. Those should not base off on the string name (in blue colour). Please return them as they were.